Libraries/Animated/src/NativeAnimatedModule.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+type EndResult = {finished: boolean};
+type EndCallback = (result: EndResult) => void;
+
+export type EventMapping = {|
+  nativeEventPath: Array<string>,
+  animatedValueTag: ?number,
+|};
+
+export type AnimatedNodeConfig = {|
+  type:
+    | 'style'
```

Comment:
Hmm actually we don't yet support enums, so I'll just update this to string when I import to FB, for now... Sorry for the confusion. cc @RSNara